js/oop.js

```diff
@@ -0,0 +1,115 @@
+var FoursquareRequest = function() {
+  this.searchCount = 0;
+};
+
+var Handlebars = function() {
+
+};
+var DomAction = function(){
+
+};
+var VenueLocation = function() {
+  var self = this;
+  var boxLat = $(self).data("lat");
+  var boxLng = $(self).data("lng");
+  var boxId = $(self).data("id");
+};
+
+var search = new FoursquareRequest();
+var template = new Handlebars();
+var Dom = new DomAction();
+var requestLocation = new VenueLocation();
+
+$(".search").click(function() {
+  var where = $('.location').val();
+  var what = $('.query').val();
+  search.ajaxRequest(where, what);
+});
+FoursquareRequest.prototype.navbarFixed = function(){
+   if (this.searchCount === 1){
+     dom.createNavbar();
+   }
+};
+FoursquareRequest.prototype.ajaxRequest = function(where, what) {
+  $.ajax({
```

Comment:
Since whatever calls this method will care about when the AJAX call is done, you'll either want to accept a (success) callback, or return the Promise. Learn more about the latter here:

https://github.com/advanced-js/deck/tree/gh-pages/demos/ajax/promises#readme